datmo/task.py

```diff
@@ -0,0 +1,128 @@
+import os
+import shlex
+
+from datmo.core.controller.task import TaskController
+from datetime import datetime
+
+
+class ClientTask():
```

Comment:
File name is `task.py`, but class name is `ClientTask`?      They should probably  match

Reply:
Agreed, i'm in the process of fixing this currently. have a few local changes i'm going to push in a bit

Reply:
I had changed the name of class since there is also another Task class in core entities. We can use the class name as `Task`